<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Citire date</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>561</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_cititre">
        <property name="text">
         <string>Alege document cvs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tb_citire">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="tb_n">
          <property name="text">
           <string>n=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tb_m">
          <property name="text">
           <string>m=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>991</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Vizualizare linkage matrix</string>
    </property>
    <widget class="QTextEdit" name="input_matrix">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>991</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Grafice</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Index cel mai mare element:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lb_index_max"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_dendrograma">
          <property name="text">
           <string>Afiseaza dendrograma</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sp_nr_clusteri"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>clusteri:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_histograme">
        <property name="text">
         <string>Afiseaza histograme</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
